sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkBuilder.java

```diff
@@ -451,16 +451,19 @@ private AutoConfiguredOpenTelemetrySdk buildImpl() {
                 ? this.config
                 : DefaultConfigProperties.create(Collections.emptyMap(), componentLoader),
             componentLoader);
+    SpiHelper spiHelper = SpiHelper.create(componentLoader);
     if (fromFileConfiguration != null) {
-      maybeRegisterShutdownHook(fromFileConfiguration.getOpenTelemetrySdk());
+      OpenTelemetrySdk openTelemetrySdk = fromFileConfiguration.getOpenTelemetrySdk();
+      maybeRegisterShutdownHook(openTelemetrySdk);
+      callAutoConfigureListeners(spiHelper, openTelemetrySdk);
```

Comment:
Can we write a simple test to verify this happens?

Reply:
that turned out to be very helpful - it didn't work before...